code/controllers/subsystem/job.dm

```diff
@@ -951,6 +951,11 @@ SUBSYSTEM_DEF(job)
 	if(possible_job.has_banned_species(player.client.prefs))
 		JobDebug("[debug_prefix] Error: [get_job_unavailable_error_message(JOB_UNAVAILABLE_SPECIES)], Player: [player][add_job_to_log ? ", Job: [possible_job]" : ""]")
 		return JOB_UNAVAILABLE_SPECIES
+	if(length(player.client.prefs.read_preference(/datum/preference/text/flavor_text)) <= 10)
```

Comment:
Maybe raise it above 10 characters? Feels extremely low, at the moment.

Reply:
used to be 150